internal/proto/proto.go

```diff
@@ -0,0 +1,7 @@
+package proto
```

Comment:
@BupycHuk Are these changes related to the license?

Reply:
Hi, yes, we are getting rid of dependency from percona-toolkit, which has an GPL license